### ISSUE TITLE
Bad idea to install thrust in $HOME

### ISSUE THREAD

**Issue by metal3d** [miketheprogrammer/go-thrust#63]
Hi, 

Using Linux (Fedora 22, x64)

At first: congrat, nice library and it seems to be working as expected, so I take a longer look :) 

Is there any way to change the thrust installation path ? At this time, it set $HOME/0.7.6 and $HOME/vendor that is (excuse me for the used term) "extremly dirty". The better should be to install thrust in "$HOME/.local/lib/thrust" I guess.

**Comment by miketheprogrammer** [comment]
Yes you can change my creating a new provisioner implementation. Please
look at tutorials

**Comment by miketheprogrammer** [comment]
did you solve this
